### ISSUE TITLE
Create-and-deploy OAuth2 client application on separate server

### ISSUE THREAD

**Issue by mblomdahl** [vtepliuk/id.mkdevops.se#13]
id.mkdevops.se will feel lonely without a working client implementation for reference. So, 
1. Create a simple web-based OAuth2 client that has a *Login* button that performs login via oauthorization on id.mkdevops.se
2. Upon logging in, the end user "resource owner" should see something they granted the client application to use, e.g. their full name
3. Call the client "oauth2-client.mkdevops.se" and deploy it to a new GCP server in Finland
4. Make sure not to commit any client secrets or similar to GitHub! ;)

**Comment by mblomdahl** [comment]
@vtepliuk Tried to get something working in iPython with this;

```python

import uuid

from requests import Request

req = Request(method='GET',
              url='http://id.mkdevops.se/oauth/authorize',
              params={
                  'client_id': '634a4601e29ec2b1e70268c972714d1b',
                  'scope': 'read write',
                  'state': uuid.uuid4().get_hex(),
                  'response_type': 'token',
                  'approval_prompt': 'auto'
              }).prepare()

req.url

```

It does not work. :( Getting some error about "duplicate scope". Wtf?

**Comment by vtepliuk** [comment]
> @vtepliuk Tried to get something working in iPython with this;
> 
> ```python
> import uuid
> 
> from requests import Request
> 
> req = Request(method='GET',
>               url='http://id.mkdevops.se/oauth/authorize',
>               params={
>                   'client_id': '634a4601e29ec2b1e70268c972714d1b',
>                   'scope': 'read write',
>                   'state': uuid.uuid4().get_hex(),
>                   'response_type': 'token',
>                   'approval_prompt': 'auto'
>               }).prepare()
> 
> req.url
> ```
> 
> It does not work. :( Getting some error about "duplicate scope". Wtf?

Now fixed in pull request #22 + response_type should be 'code' instead of 'token'
